dataactbroker/helpers/fabs_derivations_helper.py

```diff
@@ -564,6 +581,7 @@ def fabs_derivations(obj, sess, state_dict, country_dict, sub_tier_dict, cfda_di
     obj['legal_entity_city_name'] = None
     obj['place_of_performance_zip5'] = None
     obj['place_of_perform_zip_last4'] = None
+    obj['place_of_performance_scope'] = None
```

Comment:
This doesn't have to be initialized, we never check it against anything. The initialization here is in case it somehow doesn't exist before we start comparing it against other things.

Reply:
So should I not include that check in the tests too?

Reply:
You set it to `None` in the derivations don't you?

Reply:
Not in every case. I can update it to do that.

Reply:
Not for every case. I can definitely do that though.